package.json

```diff
@@ -26,11 +26,17 @@
   "module": "index.js",
   "files": [],
   "devDependencies": {
+    "@babel/core": "^7.7.4",
+    "@babel/plugin-transform-modules-commonjs": "^7.7.4",
+    "babel-jest": "^24.9.0",
+    "jest": "^24.9.0",
+    "jest-watch-typeahead": "^0.4.2",
+    "prettier": "^1.19.1",
```

Comment:
At first I've wanted to use jest's inline snapshots for tests but because stylis minifies the output those weren't readable (long lines of minified css).

So basically this stays unused right now, but I would recommend enabling prettier on the codebase 😉 Can do that as part of this PR if you want to.